tests/system/python/conftest.py

```diff
@@ -0,0 +1,108 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+""" Configuration system/python/conftest.py
+
+"""
+import pytest
+import subprocess
+import os
+
+__author__ = "Vaibhav Singhal"
+__copyright__ = "Copyright (c) 2017 OSIsoft, LLC"
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+
+@pytest.fixture
+def reset_foglamp_start():
+    assert os.environ.get('FOGLAMP_ROOT') is not None
+    subprocess.run(["$FOGLAMP_ROOT/scripts/foglamp kill"], shell=True, check=True)
+    subprocess.run(["echo YES | $FOGLAMP_ROOT/scripts/foglamp reset"], shell=True, check=True)
+    subprocess.run(["$FOGLAMP_ROOT/scripts/foglamp start"], shell=True)
+    stat = subprocess.run(["$FOGLAMP_ROOT/scripts/foglamp status"], shell=True, stdout=subprocess.PIPE)
+    assert "FogLAMP not running." not in stat.stdout.decode("utf-8")
+
+
+def pytest_addoption(parser):
+    parser.addoption("--foglamp_url", action="store", default="localhost:8081",
+                     help="foglmap client api url")
+    parser.addoption("--pi_host", action="store", default="pi-server",
+                     help="PI Server Host Name/IP")
+    parser.addoption("--pi_port", action="store", default="5460",
+                     help="PI Server PORT")
+    parser.addoption("--pi_db", action="store", default="pi-server-db",
+                     help="PI Server database")
+    parser.addoption("--pi_admin", action="store", default="pi-server-uid",
+                     help="PI Server user login")
+    parser.addoption("--pi_passwd", action="store", default="pi-server-pwd",
+                     help="PI Server user login password")
+    parser.addoption("--pi_token", action="store", default="omf_north_0001",
+                     help="OMF Producer Token")
+    parser.addoption("--south_plugin_repo", action="store", default="foglamp-south-playback",
+                     help="South Repo to clone")
+    parser.addoption("--south_plugin", action="store", default="playback",
```

Comment:
we should not need `--south_plugin_repo` as this can be used with `foglamp-south-$NAME`